nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults

```diff
@@ -273,3 +273,39 @@ nuxeo.cors.allowSubdomains=true
 nuxeo.cors.supportedMethods=GET,POST,HEAD,OPTIONS,DELETE,PUT
 nuxeo.cors.urls=
 nuxeo.cors.pattern=/.*
+
+# Bulk Service Action concurrency and partitions
+nuxeo.bulk.action.attachRetentionRule.defaultConcurrency=2
+nuxeo.bulk.action.attachRetentionRule.defaultPartitions=4
+nuxeo.bulk.action.automation.defaultConcurrency=2
+nuxeo.bulk.action.automation.defaultPartitions=2
+nuxeo.bulk.action.csvExport.defaultConcurrency=2
+nuxeo.bulk.action.csvExport.defaultPartitions=2
+nuxeo.bulk.action.driveFireGroupUpdatedEvent.defaultConcurrency=2
+nuxeo.bulk.action.driveFireGroupUpdatedEvent.defaultPartitions=4
+nuxeo.bulk.action.evalInputEventBasedRule.defaultConcurrency=2
+nuxeo.bulk.action.evalInputEventBasedRule.defaultPartitions=4
+nuxeo.bulk.action.holdDocumentsAction.defaultConcurrency=2
```

Comment:
Shouldn't this go to https://github.com/nuxeo/marketplace-retention/blob/1.0_10.10/marketplace/src/main/resources/install/templates/retention/nuxeo.defaults ?

Reply:
I'd say no since the action contrib is in a retention module inside this repository (wasn't externalized yet in 10.10).

Reply:
yeah it is easier to put the conf and processor on the same repo